packages/containers/psammead-timestamp-container/src/__snapshots__/index.test.jsx.snap

```diff
@@ -110,7 +110,7 @@ exports[`Timestamp should render without a leading zero on the day 1`] = `
 
 <time
   class="c0"
-  datetime="2018-07-07"
+  datetime="۲۰۱۸-۰۷-۰۷"
```

Comment:
datetime is not supposed to be localised https://github.com/bbc/psammead/issues/651